docs/documentation/release-notes.md

```diff
@@ -167,6 +179,15 @@ this, you may comment out the entries by using the following syntax:
 -->
 ```
 
+## demisto-sdk doc-review
+`demisto-sdk` includes the `doc-review` command to assist with the doc review process. It will check the spelling of the release notes and provide guidance if not using one of our standardized templates. Example usage:
```

Comment:
```suggestion
`demisto-sdk` includes the `doc-review` command to assist with the doc review process. It will check the spelling of the release notes and provide guidance if you are not using one of our standardized templates. Example usage:
```